recipe/meta.yaml

```
@@ -6,8 +6,8 @@ package:
   version: {{ version }}
 
 source:
-  url: http://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
-  sha256: 532c5387270dd55ff1e676e929a77c5910f2fba92fa62885b1f5ed0aef6fdde8
+  url: https://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
+  sha256: 587faa6556cc54ccd89ad35421461b4761d7809bc17a2e72f5034daea142232b
   folder: source
```

@Kwaizer Please download the following file to the repository under `recipe/`

[0001-fallback-to-readlink-if-macOS-missing-greadlink.patch.txt](https://github.com/user-attachments/files/16726009/0001-fallback-to-readlink-if-macOS-missing-greadlink.patch.txt)

then rename it to `0001-fallback-to-readlink-if-macOS-missing-greadlink.patch` and add it to version control along with the following change:

```suggestion
  url: https://hepmc.web.cern.ch/hepmc/releases/{{ name }}-{{ version }}.tar.gz
  sha256: 587faa6556cc54ccd89ad35421461b4761d7809bc17a2e72f5034daea142232b
  folder: source
  patches:
    - 0001-fallback-to-readlink-if-macOS-missing-greadlink.patch  # [osx]
```

This patch should get things working again. c.f. https://gitlab.cern.ch/hepmc/HepMC3/-/merge_requests/360 for more information.